pkg/kubelet/kubelet.go

```diff
@@ -974,6 +976,42 @@ func (kl *Kubelet) syncNodeStatus() {
 	}
 }
 
+// relabelVolumes relabels SELinux volumes to match the pod's
+// SELinuxOptions specification. This is only needed if the pod uses
+// hostPID or hostIPC. Otherwise relabeling is delegated to docker.
```

Comment:
curiosity: if we feel like we know how to do the labelling ourselves, why should we ever defer to docker?  Isn't our code net simpler, less coupled, and more fixable if we just do it ourselves?